docs/send-data/sumo-logic-distribution-for-opentelemetry-collector/fips.md

```diff
@@ -0,0 +1,188 @@
+---
+id: fips
```

Comment:
Where did this file come from ? I cannot see a Fips documentation in the Otel Git Repo.

Reply:
It's here https://github.com/SumoLogic/sumologic-otel-collector/blob/v0.71.0-sumo-0/docs/fips.md

Reply:
@angadrandhawa1 it's in their TOC as well.
<img width="731" alt="Screenshot 2023-02-21 at 10 32 29 AM" src="https://user-images.githubusercontent.com/56411016/220429949-e6da1b1a-2c5c-4819-b017-3eca2ba6e19e.png">